README.md

```diff
@@ -17,8 +17,8 @@ phpunit-mock-objects without the World ending.
 Mockery is released under a New BSD License.
 
 The current released version on Packagist is `0.9.4`.
-The current released version for PEAR is `0.9.0`. Composer users may instead opt to use
-the current master branch aliased to `1.0.x-dev`.
+Composer users may opt to use the current master branch aliased to `1.0.x-dev`.
```

Comment:
No? They either want `0.9.*` or `~1.0`. No need for any `dev-master` or anything else.